help/en/package/jmri/jmrit/withrottle/UserInterface.shtml

```diff
@@ -22,33 +22,15 @@
 
   <div class="nomenu" id="mBody">
     <div id="mainContent">
-      <h1>JMRI: WiFi Connected Throttle</h1>The WiThrottle window controls the
+      <h1>JMRI: WiFi Connected Throttle</h1>
+      JMRI provides a "WiThrottle Server" which can provide a bridge between
+      the connected DCC system and WiFi-based throttles.
+      
+      <p>The WiThrottle Server window controls the
       connection between WiFi throttles, e.g. an iPhone, iPod
-      Touch, or Android Phone, and JMRI.
-
-      <p>It can be set to start automatically with the "<a href=
-      "../../../apps/TabbedPreferences.shtml#Startup">Start Up</a>"
-      section in the "<a href=
-      "../../../apps/TabbedPreferences.shtml">JMRI
-      Preferences</a>".</p>
+      Touch, or Android Phone, and JMRI.  
 
-      <p><a href=
-      "../../../apps/TabbedPreferences.shtml#WiThrottle">Preferences</a>
-      for the WiFi throttles can be found in the JMRI Preferences
-      under the heading WiThrottle.</p>
-
-      <p>This utility broadcasts its connection information across
-      all local networks, so it can be picked up by any device
-      looking for the service. This eliminates having to enter
-      connection information into the device. If the computer
-      running JMRI is protected by a software firewall, you may
-      need to add an exception for JMRI or disable the firewall
-      to allow the WiFi throttle to connect.</p>
-
-      <p>When using a WiFi throttle, if the computer enters sleep
-      mode, control will be lost. System preferences may need to be
-      adjusted to prevent the computer from putting itself to sleep
-      when it is inactive. <a href=
```

Comment:
@devel-bobm not sure I follow why lines 35-51 were removed. Can you explain?

Reply:
Nope.  I'll see that they get returned...